--- responses-0.17.0/responses/__init__.py.orig	2022-01-07 17:17:57.000000000 +0100
+++ responses-0.17.0/responses/__init__.py	2022-03-10 18:30:33.217435207 +0100
@@ -141,14 +141,14 @@ def _ensure_str(s):
 
 
 def _cookies_from_headers(headers):
-    try:
+    if six.PY3:  # http.cookies.SimpleCookie().load() from future fails with unicode input
         import http.cookies as _cookies
 
         resp_cookie = _cookies.SimpleCookie()
         resp_cookie.load(headers["set-cookie"])
 
         cookies_dict = {name: v.value for name, v in resp_cookie.items()}
-    except (ImportError, AttributeError):
+    else:
         from cookies import Cookies
 
         resp_cookies = Cookies.from_request(_ensure_str(headers["set-cookie"]))
